pom.xml

```diff
@@ -421,6 +421,7 @@ limitations under the License.
                         <exclude>**/.*/**</exclude>
                         <!-- Generated content -->
                         <exclude>**/target/**</exclude>
+                        <exclude>**/*.log</exclude>
```

Comment:
What is the reason for the generation of these logs?

Reply:
`Derby.log` will be generated when using idea for paimon sink local testing. 
`flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/derby.log`